dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/FATSuite.java

```diff
@@ -70,11 +77,31 @@ public class FATSuite {
                     .andWith(FeatureReplacementAction.EE10_FEATURES().conditionalFullFATOnly(FeatureReplacementAction.GREATER_THAN_OR_EQUAL_JAVA_21))
                     .andWith(FeatureReplacementAction.EE11_FEATURES());
 
+    //Server used for setup
+    private static LibertyServer server = LibertyServerFactory.getLibertyServer("globalTLDServer");
+    
+    public static final String USER_FEATURE_PATH = "usr/extension/lib/features/";
+    public static final String USER_BUNDLE_PATH = "usr/extension/lib/";
+    public static final String USER_FEATURE_MF_FAT_PATH = "features/globaltld-1.0.mf";
+    public static final String USER_FEATURE_NAME = "globaltld-1.0.mf";
+    public static final String USER_BUNDLE_JAR_FAT_PATH = "bundles/io.openliberty.test.tld.jar";
+    public static final String USER_BUNDLE_JAR_NAME = "io.openliberty.test.tld.jar";
+
     /**
      * @see {@link FatLogHandler#generateHelpFile()}
      */
     @BeforeClass
-    public static void generateHelpFile() {
+    public static void setup() throws Exception {
+
+        // Install user feature 
+        // TODO: Transform the jar to work with EE9+ features or recreate this test in the other Pages FATs
```

Comment:
Do you have an issue opened on the back log for this?

Reply:
https://github.com/OpenLiberty/open-liberty/issues/27345